packages/configurationmap/components/ConfigurationMap.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+// import { style } from "../style";
+import { Table, Column } from "../../";
+
+export interface ConfigurationMapProps {
+  data: { [key: string]: string };
+}
+
+const cellRenderer = column => data => <span>{data[column]}</span>;
+
+class ConfigurationMap extends React.PureComponent<ConfigurationMapProps, {}> {
+  public render() {
+    const { data } = this.props;
+
+    const tableData = Object.entries(data);
+
+    return (
+      <Table data={tableData}>
+        <Column header="" cellRenderer={cellRenderer(0)} />
+        <Column header="" cellRenderer={cellRenderer(1)} />
```

Comment:
Why do we have two rows hard-coded?

Reply:
A Configuration map is this in DC/OS 
![image](https://user-images.githubusercontent.com/156010/58247654-7382ee80-7d5a-11e9-9748-c7feee522edb.png)
The most minimal approach to this is a table with 2 Columns. This is why there are 2 Columns hard coded.

Reply:
But still, having a custom number of rows wouldn't be much less minimal and it would be useful.

Reply:
Could you give me an example? I think I am misunderstanding your intend.

Reply:
In practice we can have a different number of rows and columns. So for this component to be useful, it should support that.

Reply:
I will discuss this with design, to come to an agreement. Currently I would suggest to do it only with 2 columns since this will cover most of the use cases. Since this is a minimal implementation this should be enough ground covered.

Reply:
@GeorgiSTodorov Where do we have multiple number of columns? Wouldn't this be more of a table than a config map?

Reply:
@DanielMSchmidt , I don't know if we have more than two anywhere, but I can imagine situations where we want to have more than two. I want our minimal component to be usable in every case.

Reply:
I think if we try to build something that works for all use cases we will spend time that is not needed. Let's iterate on it and if we have a config map with more then a key value relationship we can see if it's really a config map or a table or a special new component

Reply:
Okay, agreed.